pkgs/servers/sql/postgresql/ext/citus.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, postgresql, curl, lz4 }:
+
+stdenv.mkDerivation rec {
+  pname = "citus";
+  version = "11.3.0";
+
+  src = fetchFromGitHub {
+    owner  = "citusdata";
+    repo   = "citus";
+    rev    = "v${version}";
```

Consider using the `finalAttrs` pattern

```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "citus";
  version = "11.3.0";

  src = fetchFromGitHub {
    owner  = "citusdata";
    repo   = "citus";
    rev    = "v${finalAttrs.version}";
```